source/includes/language-compatibility-table-php.rst

```diff
@@ -13,8 +14,19 @@
      - PHP 7.0
      - PHP 5.6
      - PHP 5.5
-
+   * - ext + lib 1.15
```

Comment:
Issue: We should probably keep a footnote similar to the current one to explain that 1) we skipped lib 1.14 and 2) the versions are now equivalent between lib and ext. I can update the footnote text in a different PR.

Suggestion:

```suggestion
   * - ext + lib 1.15 [#PHP1.15-version-parity]_
```